functions.php

```diff
@@ -39,4 +39,23 @@ function displayData(array $allItems): string
         }
     }
     return $output;
-}
\ No newline at end of file
+}
+
+/**
+ * @param string $latin user input for the latin name of a new item
+ * @param string $common user input for the common name of a new item
+ * @param int $height user input for the height of a new item
+ * @param int $width user input for the width of a new item
+ * @param string $death user input for the deathly cat. of a new item
+ */
+function newItem(string $latin, string $common, int $height, int $width, string $death)
+{
+    $db = collectionDbConnection();
```

Comment:
I would move this function call to line 9 of index.php instead of having functions calling functions